x-pack/test/security_solution_endpoint/apps/endpoint/policy_list.ts

```diff
@@ -101,5 +111,42 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         expect(policyTotal).to.equal('0 Policies');
       });
     });
+
+    describe('and user clicks on page header create button', () => {
+      beforeEach(async () => {
+        await pageObjects.policy.navigateToPolicyList();
+        await (await pageObjects.policy.findHeaderCreateNewButton()).click();
+      });
+
+      it('should redirect to ingest management integrations add datasource', async () => {
+        await pageObjects.ingestManagerCreateDatasource.ensureOnCreatePageOrFail();
+      });
+
+      it('should redirect user back to Policy List if Cancel button is clicked', async () => {
+        await (await pageObjects.ingestManagerCreateDatasource.findCancelButton()).click();
+        await pageObjects.policy.ensureIsOnPolicyPage();
+      });
+
+      it('should redirect user back to Policy List if Back link is clicked', async () => {
+        await (await pageObjects.ingestManagerCreateDatasource.findBackLink()).click();
+        await pageObjects.policy.ensureIsOnPolicyPage();
+      });
+
+      it('should display custom endpoint configuration message', async () => {
```

Comment:
@parkiino FYI. We spoke about this last week. In this PR, I added a check that ensures our custom configuration message is used on the Create Datasource page of Ingest. We still need to write up some tests for the Edit case in ingest.

Reply:
gotcha